draft-ietf-cbor-7049bis.md

```diff
@@ -757,9 +757,9 @@ string item of the same major type, the string is not well-formed.
 
 If any definite-length text string inside an indefinite-length text
 string is invalid, the indefinite-length text string is invalid.  Note
-that this implies that the bytes of a single UTF-8 character cannot be
-split up between chunks: a new chunk of a text string can only be
-started at a character boundary.
+that this implies that the bytes of a single UTF-8 code point cannot be
```

Comment:
This works if you are well-versed in Unicode, not so much if you aren't.
This wording suggests the existence of a "UTF-8 code point".
Maybe:

> the UTF-8 bytes of a single Unicode code point (scalar value)

(Compare the use of "code point" in RFC 3629, which is the reference for UTF-8.)

(The other changes are fine.)

Reply:
Sure, done.